backend/doc/cli.md

```diff
@@ -65,24 +159,25 @@ optional arguments:
                         application config file to read (default:
                         development.ini)
   -d, --debug_mode      enable Tracim log for debug
-  -w, --delete-owned-sharespaces
-                        delete also owned sharespaces of user
-  -a, --anonymize-if-required
-                        anonymizes the user where he cannot be deleted
-  -r, --delete-all-user-revisions
-                        this allow to delete all user revisions. Warning !
-                        This may create inconsistent database
+  --dry-run             dry-run mode, simulate action to be done but do not
+                        modify anything
   -b, --best-effort     trying doing the best deletion possible, same as '-w
                         -a'
   -f, --force           force user deletion, same as '-r -w'. Warning ! This
                         may create inconsistent database
-  --dry-run             dry-run mode, simulate action to be done but do not
-                        modify anything
-  -l LOGINS [LOGINS ...], --login LOGINS [LOGINS ...]
-                        user logins (email) to delete one or more user
+  -a, --anonymize-if-required
+                        anonymizes the user where he cannot be deleted
   --anonymize-name ANONYMIZE_NAME
                         anonymized user display name to use if anonymize
                         option is activated
+  -r, --delete-all-user-revisions
+                        this allow to delete all user revisions. Warning !
+                        This may create inconsistent database
```

Comment:
```suggestion
                        delete all user revisions. Warning!
                        This may put the database into an inconsistent state
```